exercises/G-conditionals-2/exercise-2.js

```diff
@@ -7,7 +7,13 @@
 
 */
 
-function studentPassed(grade) {}
+function studentPassed(grade) {
+  if (grade >= 50) {
+    return "passed";
+  } else if (grade <= 50) {
+    return "failed";
+  }
```

Comment:
is says `if the grade is less than 50` - so we can just say `grade < 50` instead of `<=`

even better we can just use a plain else statement because it the result is inclusive (everything that doesn't match the first argument will match the second:

```
 if (grade >= 50) {
     return "passed";
   } else {
     return "failed";
   }
```

even better we don't need the else at all, because the `return` on line 12 will terminate the function if it matches, so we can reduce this all down to:

```
 if (grade >= 50)  return "passed"
 return "failed"
```

or if you're being extra fancy go for a ternary and fit it all on one line:
```
return (grade >= 50) ? "passed" : "failed"
```